docs/content/users/extend/traefik-router.md

```diff
@@ -54,17 +54,9 @@ Project-specific configuration is automatically generated in the project’s `.d
 
 * The `certs` directory contains the `<projectname>.crt` and `<projectname>.key` certificate generated for the project.
 * The `config/<projectname>.yaml` file contains the configuration for the project, including information about routers, services, and certificates.
-    * If you alter a `config/<projectname>.yaml` file, you need to remove the `#ddev-generated` from the top of the file, so it won't be overwritten on the next `ddev start`.
-    * It is possible to add additional files in `config` that you own and manage that add additional facilities. For example, a `config/example_middlewares.yaml` would be read by the router and used. It's important that the filename be keyed by the project name, and usually its contents need to be unique to the project. For example, if it defines middleware implementations, they should usually be limited to the project. For example, an extra file might be:
-
-      ```yaml
-      http:
-        middlewares:
-          example-redirectHttps:
-            redirectScheme:
-              scheme: https
-              permanent: true
-      ```
+    * You can modify a project's dynamic configuration by adding `dynamic_config.*.yaml` files to `.ddev/traefik/`. These will be merged with an alphanumeric sort **override** strategy, so anything that is needed can be added or replaced.
+    * An example `.ddev/traefik/dynamic_config.middlewares.yaml.example` file is provided in `.ddev/traefik/`. It can be modified to define and assign different middlewares, as well as overriding any default parameters. Please see `.ddev/traefik/README.md` file for more details.
+    * The extra config files may contain Go template placeholders, for injecting App-specific data into the config. But this is an advanced option that most can ignore. If you use them, note that template `{{ }}` placeholders cannot go at the start of a key (e.g. `{{.App.Name}}-web-80-http`). They must be preceded by any character, or used in YAML values. This means that if a middleware needs to be prefixed with the app's name, it will either need to be done manually or prefixed when an add-on creates its dynamic_config.*.yaml file from its own go template.
```

Comment:
```suggestion
    * The extra config files may contain Go template placeholders, for injecting App-specific data into the config. But this is an advanced option that most can ignore. If you use them, note that template `{{ }}` placeholders cannot go at the start of a key (e.g. `{{.App.Name}}-web-80-http`). They must be preceded by any character, or used in YAML values. This means that if a middleware needs to be prefixed with the app's name, it will either need to be done manually or prefixed when an add-on creates its `dynamic_config.*.yaml` file from its own go template.
```